site/public/js/forum.js

```diff
@@ -1680,7 +1679,7 @@ function refreshCategories() {
         $(this).trigger('eventChangeCatClass');
     });
 
-    $('.cat-buttons').bind('eventChangeCatClass', changeColorClass);
+    $('.cat-buttons').on('eventChangeCatClass', changeColorClass);
```

Comment:
`.bind()` and `.on()` are equivalent in the current version of jQuery.  I believe that the `.on()` is slightly preferred, but all current usages of `.bind()` should be converted to `.on()` and an eslint rule prohibiting `.bind()` should be added if you want to make this change.